backend/routes/base/providers.js

```diff
@@ -5,7 +5,10 @@ var buildStatic = function(db, router){
 
 var buildDynamic = function(db, router, auth, forumClient){
     
+    let log = require('npmlog');
```

Comment:
I'd change it to `const` if it's not going to be reassigned.

Reply:
Agreed